src/main/java/uk/gov/hmcts/reform/laubackend/cases/controllers/CaseSearchController.java

```diff
@@ -144,15 +159,22 @@
             verifyRequestSearchParamsAreNotEmpty(inputParamsHolder);
             verifyRequestSearchParamsConditions(inputParamsHolder);
 
+            final long timeStart = System.currentTimeMillis();
             final CaseSearchGetResponse caseSearch = caseSearchService.getCaseSearch(inputParamsHolder);
-
```

Comment:
Did you mean to exlude the verification above from the timeStart

Reply:
I did but now I've moved the start date before the verify.